packages/react/src/patterns/sections/CardSection/__stories__/data/cards.json

```diff
@@ -52,8 +52,8 @@
         "defaultImage": "https://picsum.photos/id/1003/1056/480",
         "alt": "cards with image"
       },
-      "title": "Topic",
-      "copy": "Natural language processing.",
+      "eyebrow": "Topic",
+      "heading": "Natural language processing.",
```

Comment:
Need to rename `title` to heading above I believe.

Reply:
I changed `title to heading` here I believe.

Reply:
@photodow: when I asked you said, Blog is eyebrow so I have the title to eyebrow.
![image](https://user-images.githubusercontent.com/53789187/74546308-63c6be00-4f18-11ea-9c66-2a099a5d86d6.png)

Reply:
Sorry for the confusion here, I believe I was referring to the rest of the data above. You made changes to CardSection, and I don't think the others will work anymore.

Reply:
> @photodow: when I asked you said, Blog is eyebrow so I have the title to eyebrow.

Yeah, Blog here is the eyebrow. But AI's Interface Revolution is the title/heading.

Reply:
yeah that is I made I believe.
AI's Interface Revolution  is heading
Blog , Topic is eyebrow.